scipy/spatial/qhull.pyx

```diff
@@ -2410,3 +2449,269 @@ class Voronoi(_QhullUser):
             self._ridge_dict = dict(zip(map(tuple, self.ridge_points.tolist()),
                                         self.ridge_vertices))
         return self._ridge_dict
+
+#------------------------------------------------------------------------------
+# Halfspace intersection interface, for Python
+#------------------------------------------------------------------------------
+
+class _QhalfUser(object):
```

Comment:
Indeed, the only point I wanted to check was how the incremental mode works with halfspace intersection... I haven't really had time to do so.